frontend/src/metabase/admin/settings/utils.spec.js

```diff
@@ -0,0 +1,20 @@
+import { prepareAnalyticsValue } from './utils'
+
+describe('prepareAnalyticsValue', () => {
+    const defaultSetting = { value: 120, type: 'number' }
+
+    const checkResult = (setting = defaultSetting, expected = "success") =>
```

Comment:
It's nice when you can look at the body of a test and see roughly what it's testing